scripts/install_third_party_libs.py

```diff
@@ -30,11 +30,11 @@
 # These libraries need to be installed before running or importing any script.
 
 PREREQUISITES = [
-    ('pyyaml', '5.4.1', os.path.join(TOOLS_DIR, 'pyyaml-5.4.1')),
+    ('pyyaml', '6.0', os.path.join(TOOLS_DIR, 'pyyaml-6.0')),
```

Comment:
Just to confirm, the major update in pyyaml isn't causing any issue for us, right? (I think we use pyyaml for loading demo exploration.)

Reply:
No, it isn't.